common/siklu/siklu_nfs_boot.c

```diff
@@ -18,15 +18,15 @@
 #define USB_RESET_GPIO_PIN 	55
 
 static int nfs_get_file(const char *path, const char *file, char *address) {
-	char cmd[1024];
+	static char cmd[1024];
 	
 	snprintf(cmd, sizeof(cmd), 
 			"nfs \"%s\" \"%s:%s/%s\"", address, getenv(ENV_NFS_SERVERIP), path, file);
 
 	return run_command(cmd, 0);
 }
 
-int format_rootpath_and_developer_id(const char *rootpath, const char *developer_id, 
+static int format_rootpath_and_developer_id(const char *rootpath, const char *developer_id, 
```

Comment:
This change looks unrelated

Reply:
You are right. I will split it into another PR.